snuba/datasets/errors_replacer.py

```diff
@@ -314,6 +314,32 @@ def process_exclude_groups_and_replacement_types_results(
     return needs_final, exclude_groups, replacement_types
 
 
+def get_latest_replacement_time_by_projects(
+    project_ids: Sequence[int], state_name: Optional[ReplacerState]
+) -> Optional[float]:
```

Comment:
please return a datetime. It will be much easier and safe to manage than a bare float. 
It also makes the interface a lot more intuitive. If you see a float in the return value you don't know what it is an you need to read through the code.
If you return a date time you immediately know what it is.